node/pkg/terra/watcher.go

```diff
@@ -156,6 +156,11 @@ func (e *Watcher) Run(ctx context.Context) error {
 	readiness.SetReady(e.readiness)
 
 	go func() {
+		latestBlockURL := fmt.Sprintf("%s/blocks/latest", e.urlLCD)
```

Comment:
should we add this logic to the watcher struct like `contractAddressFilterKey` and `contractAddressLogKey` so that all the chain differing logic stays in one place?